com.unity.shadergraph/Editor/Importers/ShaderGraphImporter.cs

```diff
@@ -338,10 +338,14 @@ internal static string GetShaderText(string path, out List<PropertyCollector.Tex
         }
 
 #if VFX_GRAPH_10_0_0_OR_NEWER
-        // TODO: Fix this
+        // TODO: Fix this - VFX Graph can now use ShaderGraph as a code generation path. However, currently, the new
```

Comment:
Currently we must still support the ShaderGraphVfxAsset to maintain compatibility with the Visual Effect target. Thus this PR is also still slightly dependent on some of the information kept within this container (fragment/vertex properties). We also rely on it to filter out only shader graphs that support VFX in the object selector.